src/identity.cc

```diff
@@ -167,6 +172,27 @@ ModelState::SupportsFirstDimBatching(bool* supports)
   return nullptr;  // success
 }
 
+TRITONSERVER_Error*
+ModelState::CreationDelay()
+{
+  // Feature for testing purpose...
+  // look for parameter 'creation_delay_sec' in model config
+  // and sleep for the value specified
+  common::TritonJson::Value parameters;
+  if (model_config_.Find("parameters", &parameters)) {
+    common::TritonJson::Value creation_delay_sec;
+    if (parameters.Find("creation_delay_sec", &creation_delay_sec)) {
+        std::string creation_delay_sec_str;
+        RETURN_IF_ERROR(creation_delay_sec.MemberAsString("string_value", &creation_delay_sec_str));
+        LOG_MESSAGE(
+      TRITONSERVER_LOG_INFO,
```

Comment:
The format seems wrong. Can you run formatter.

Reply:
Done